src/hooks/__tests__/useDelayedDoubleClick.test.js

```diff
@@ -18,7 +18,7 @@ describe('hooks/useDelayedDoubleClick', () => {
 
     expect(clickFunc).not.toHaveBeenCalled();
 
-    await resolveAfterMs(DOUBLE_CLICK_TIMEOUT);
+    await resolveAfterMs(DOUBLE_CLICK_TIMEOUT * 2);
```

Comment:
No, this won't solve the issue. Only circumvent the problem.
See #1146 .

Reply:
Great work!